### DISCUSSION TITLE
Is mysql field type varbinary available?

### DISCUSSION THREAD

**Discussion by sunny-kevin** [laravel/framework#48490] [Q&A] [answered]
hello, I wanna use mysql field type `varbinary ` for migration，but I don't see the matching type of `varbinary`，please help, thanks.

**Comment by shaedrich** [comment, marked as answer]
This seems to be only available for MS SQL: https://github.com/laravel/framework/blob/ef76b3a4dc0616960a1532c29cad3b8c429f8597/src/Illuminate/Database/Schema/Grammars/SqlServerGrammar.php#L806-L809

Taylor Otwell suggests doing this via a raw query: https://github.com/laravel/framework/issues/3297#issuecomment-33505183

There seems to be a workaround, however, I haven't tested it and it is somewhat old: https://github.com/laravel/framework/issues/1606#issuecomment-650633301

**Reply by sunny-kevin** [reply]
thanks.

**Reply by shaedrich** [reply]
You're welcome.
